### PR TITLE
[14.0][IMP] partner_contact_access_link: Open base.view_partner_form.

### PR DESCRIPTION
**Pull request by bobslee**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please don't modify by hand README.md (it's done automatically) and the manifest version (it's done as well on merge). Why do you want to make this change?

**Comment by bobslee** [comment]
Thanks for responding @pedrobaeza .

Ok. Forgive my lack of experience.
Are these updates (README, manifest version) mentoined somewhere in OCA guidelines?

We need this change, to get the chatter in the opened form-view.
So the link (action) now opens the "base.view_partner_form".

Can you agree or explain why it's undesireable?

**Comment by pedrobaeza** [comment]
I would argue the contrary: is any place where it puts that you have to change such things? On contrary, you can see this line in the README.md:

Selección_008

Please undo that and fix pre-commit stuff.

**Comment by bobslee** [comment]
Done. Sorry, I just didn't spotted in a rush.

**Review by pedrobaeza — APPROVED** [review]

File: partner_contact_access_link/README.rst [hunk 1]
```diff
@@ -71,6 +71,7 @@ Contributors
 * Jairo Llopis (`Tecnativa <https://www.tecnativa.com/>`__)
 * Harald Panten (`Sygel Technology <https://www.sygel.es/>`__)
 * Tharathip Chaweewongphan <tharathipc@ecosoft.co.th>
+* Bob Leers (`Nova Code <https://www.novacode.nl/>`__)
```
Comment (emagdalenaC2i) [inline partner_contact_access_link/README.rst]: Remove the changes in the README file.
You should change the contributors file instead as is mentioned here https://github.com/OCA/maintainer-tools/tree/master/template/module

**Review by emagdalenaC2i — COMMENTED** [review]
Please see the comments and squash your commits

File: partner_contact_access_link/README.rst (hunk 1 above)
Comment (bobslee) [reply partner_contact_access_link/README.rst]: Done @emagdalenaC2i 
Let me know if any issues.

Comment (emagdalenaC2i) [reply partner_contact_access_link/README.rst]: @bobslee Sorry but I can't see the difference with this change
Please, could you explain it?

Comment (bobslee) [reply partner_contact_access_link/README.rst]: @emagdalenaC2i I squashed the commits. The one you mentoined is "outdated" (see it's a label).
So there should now be only 1 commit: 4f2ab4ef88e1fe16e5c9a1a4adc391f9dde92cac

**Pull request closed without merging**